antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlLexer.g4

```diff
@@ -350,8 +350,8 @@ LOCK
     : L O C K
     ;
 
-MERGE
-    : M E R G E
+COMPACT
```

Comment:
In the lexer, keywords are sorted lexicographically, please put this keyword in the correct position.